.circleci/config.yml

```diff
@@ -117,33 +179,38 @@ workflows:
   build:
```

Comment:
Does the workflow version need to be upped as well?

Reply:
Just the circle version [https://circleci.com/orbs/registry/orb/circleci/windows] based on my understanding, to use orbs/ windows, this needs to be upped.